homeassistant/components/steam_online/sensor.py

```diff
@@ -104,6 +109,7 @@ def update(self):
         try:
             self._profile = self._steamod.user.profile(self._account)
             self._game = self._get_current_game()
+            self._game_id = self._profile.current_game[0]
```

Comment:
Does `self._profile.current_game[0]` return `None` when you are not currently playing a game?

Reply:
I am not 100% sure to be honest ...
I never bothered checking it since the call to self._get_current_game() (line above) gets self._profile.current_game[0] ... so I assumed that this isn't a dodge try catch to handle that.. will check that out tomorrow.

Reply:
Any updates on this? :)

Reply:
Sorry ... had other commitments crop up :( ... checking now :)

Reply:
Yes it does .. I thought that the "self._profile.current_game" may have been failing since it may be None when not playing (would have been swallowed by the exception if this was happening)... but instead it contains 3 None's :) ... so looks all good 👍

Reply:
Perfect thanks for looking into it :)